core/amber/src/main/python/core/runnables/test_network_receiver.py

```diff
@@ -46,7 +47,7 @@ def data_payload(self):
             'Brand': ['Honda Civic', 'Toyota Corolla', 'Ford Focus', 'Audi A4'],
             'Price': [22000, 25000, 27000, 35000]
         }, columns=['Brand', 'Price'])
-        return DataFrame(frame=[Tuple(r) for _, r in df_to_sent.iterrows()])
+        return OutputDataFrame(frame=[ImmutableTuple(r, ['Brand', 'Price']) for _, r in df_to_sent.iterrows()], schema=['Brand', 'Price'])
```

Comment:
from this I feel this schema design can be revisited. now `OutputDataFrame(frame, schema)` and `ImmutableTuple(data, schema)` there are two copies of schemas. can we unify them?

Reply:
decided to remove `ImmutableTuple` and a schema is not currently bonded with a `Tuple`. will revisit this later.